components/kubeconfig-service/Dockerfile

```
@@ -17,26 +17,25 @@ COPY . .
 RUN go build -v -o main ./cmd/generator/main.go
 RUN mkdir /app && mv ./main /app/main
 
-FROM eu.gcr.io/kyma-project/external/alpine:3.16.2
-LABEL source = git@github.com:kyma-project/control-plane.git
+
+FROM eu.gcr.io/kyma-project/external/alpine:3.16.2 as alpine
 
 WORKDIR /app
 
 #
 # Install certificates
 #
 RUN apk --no-cache add --update openssl zlib busybox --repository=https://dl-cdn.alpinelinux.org/alpine/edge/main
-
 RUN apk add --no-cache ca-certificates
 
-#
+FROM scratch
+LABEL source = git@github.com:kyma-project/control-plane.git
+WORKDIR /
 # Copy binary
-#
-
 COPY --from=builder /app /app
-
-#
+COPY --from=alpine /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=alpine /usr/bin/openssl /usr/bin/
+COPY --from=alpine /bin/busybox /bin/
+COPY --from=alpine /usr/lib/* /usr/lib/
```

```suggestion
COPY --from=alpine /usr/bin/openssl /usr/bin/
COPY --from=alpine /bin/busybox /bin/
COPY --from=alpine /usr/lib/* /usr/lib/
```

I believe we won't need these.

---

Ok i'll remove them, i just kept them as the original image.